contrib/basicauth/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+nginx:
+  build: "nginx"
+  ports:
+    - "5441:5441"
+  links:
+    - registryv2:registryv2
+registryv2:
+  build: "../../"
+  ports:
+    - "5000"
```

Comment:
@moxiegirl @thaJeztah Can we try this without exposing the port? I think that should work.

@moxiegirl I am a little shaky in understanding this area, as well. If you want, we can sit down and try to understand why that would work when containers are linked.